FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,48 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url) { [weak self] result in
+			guard self != nil else {return}
+			switch result {
+			case .success((let data, let response)):
+				completion(FeedItemsMapper.map(data, from: response))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+private struct Root: Decodable {
+	let items: [Item]
+	
+	var feedImages: [FeedImage] {
+		items.map { $0.feedImage }
+	}
+}
+
+private struct Item: Decodable {
+	let image_id: UUID
+	let image_desc: String?
+	let image_loc: String?
+	let image_url: URL
+	
+	var feedImage: FeedImage {
+		FeedImage(id: image_id,
+				  description: image_desc,
+				  location: image_loc,
+				  url: image_url)
+	}
+}
+
+internal final class FeedItemsMapper {
```

Comment:
`FeedItemsMapper` is a RemoteFeedLoader implementation detail, it doesn't need to be `internal`

Please make it `private`
```suggestion
private final class FeedItemsMapper {
```